3p/iframe-messaging-client.js

```diff
@@ -25,6 +25,8 @@ export class IframeMessagingClient {
   constructor(win) {
     /** @private {!Window} */
     this.win_ = win;
+    /** @private {?string} */
+    this.rtvVersion_ = (win.AMP_CONFIG && win.AMP_CONFIG.v) || null;
```

Comment:
it's a timestamp (13 digits) prefixed with 2 digits. However, we have plan to change the format (we might add more digits).

So at this point, maybe you can assume its a string of 15 - 20 digits. 

The script URL looks like this:
```
https://cdn.ampproject.org/rtv/001481677709953/amp4ads-host-v0.js
```